__tests__/vfs.js

```diff
@@ -29,15 +29,13 @@ const otherMount = {
   }
```

Comment:
This file should not have changes.

Reply:
> This file should not have changes.



I didn't change it, you changed it to create a search test, you told me to rebase it, it was added, I didn't change it

Reply:
But this PR should **not contain this**. It's already in master. This is stuff you commited.